src/modular/bernstein_yang.rs

```diff
@@ -86,7 +86,7 @@ impl<const SAT_LIMBS: usize, const UNSAT_LIMBS: usize>
         // integer to be inverted and the modulus the inverter was created for.
         // Thus, if "f" is neither 1 nor -1, then the sought inverse does not exist.
         let antiunit = f.eq(&Int62L::MINUS_ONE);
-        let ret = self.norm(d, antiunit);
+        let ret = self.norm(d, ConstChoice::from_word_lsb(antiunit as Word));
```

Comment:
It would be nice to make `eq` return `ConstChoice` here but I'm saving that as a followup